src/ngx_lua_resty_lmdb_module.h

```diff
@@ -50,4 +52,7 @@ extern ngx_module_t ngx_lua_resty_lmdb_module;
 #endif
 
 
+#define EVP_DIGEST_CONSTANT "1qazxsw2"
```

Comment:
```suggestion
#ifndef EVP_DIGEST_CONSTANT
#define EVP_DIGEST_CONSTANT "1qazxsw2"
#endif
```

so we can update the salt at compile time